app/src/components/TeamContracts.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <div id="team-contracts" class="overflow-x-auto">
+    <table class="table">
+      <!-- head -->
+      <thead>
+        <tr>
+          <th></th>
+          <th>Type</th>
+          <th>Contract Address</th>
+          <th>Admins</th>
+          <th>Details</th>
+          <th>Events</th>
+          <!-- Added a new column for contract details -->
+        </tr>
+      </thead>
+      <tbody>
+        <!-- row 1 -->
+        <tr v-for="(contract, index) in _contracts" :key="index" class="bg-base-200">
+          <th>{{ index + 1 }}</th>
+          <td>{{ contract.type }}</td>
+          <td><AddressToolTip :address="contract.address" class="text-xs" /></td>
+          <td>
+            <button @click="openAdminsModal(contract)" class="btn btn-ghost btn-xs">
```

Comment:
```suggestion
            <button @click="openAdminsModal(contract)" class="btn btn-ghost btn-xs" data-test="open-admin-modal-btn">
```


by adding a data test attribute like that you can simply select the component you want for the test